packages/graphback-runtime/src/service/CRUDService.ts

```diff
@@ -1,14 +1,17 @@
-import { GraphbackOperationType, upperCaseFirstChar } from "@graphback/core"
+import { GraphbackOperationType, upperCaseFirstChar, GraphbackCRUDGeneratorConfig } from "@graphback/core"
 import * as DataLoader from "dataloader";
-import { GraphQLObjectType } from 'graphql';
 import { PubSubEngine } from 'graphql-subscriptions';
 import { GraphbackDataProvider } from "../data/GraphbackDataProvider";
 import { defaultLogger, GraphbackMessageLogger } from '../utils/Logger';
 import { GraphbackPage } from "../GraphbackPage"
 import { GraphbackOrderBy } from '../GraphbackOrderBy';
 import { GraphbackCRUDService, ResultList } from "./GraphbackCRUDService";
-import { GraphbackPubSub } from "./GraphbackPubSub"
 
+export interface CRUDServiceConfig {
```

Comment:
After seeing complains in issues I think we need to document PubSub usage. any comment will work here